utils/ort/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -288,6 +261,33 @@ object CopyrightStatementsProcessor {
      * collapsed. The returned [Result] contains successfully processed as well as unprocessed statements.
      */
     fun process(copyrightStatements: Collection<String>): Result {
+        /**
```

Comment:
Personally I'm not a fan of using this nesting heavily, IMO it decreases readability as
levels of abstraction become mixed again.